api/src/controllers/publications.js

```diff
@@ -154,7 +156,7 @@ async function getGoogleScholarPublications(req, res) {
     playwrightConfig.pageSizeSuffix +
     pageSize +
     playwrightConfig.sortBySuffix;
-  console.log(url);
+  logger.info(url);
```

Comment:
Maybe add some more context to the log statement?
e.g. `GScholar Publications for user xxx = ${url}`